spec/visual/templates/layout.erb

```diff
@@ -38,18 +38,164 @@
     #page-footer {
       border-top: 1px dotted
     }
+
+    #theme-selector {
+      display: inline-block;
+      position: relative;
+      margin: 0 auto;
+      width: 170px;
+      font-family: Arial;
+      font-size: smaller;
+      text-align: left;
+    }
+    #theme-selector select, .select-hide {
+      display: none;
+    }
+    #theme-selected {
+      color: <%= @comment_color %>;
+      font-weight: bold;
+      border-radius: 8px;
+      z-index: 99;
+    }
+    #theme-selected:hover:after,
+    #theme-selected.select-arrow-active:after {
+      position: absolute;
+      top: 3px;
+      right: 10px;
+      width: 0;
+      height: 0;
+      content: "";
+      border: 6px solid transparent;
+      border-color: transparent transparent currentColor transparent;
+    }
+    #theme-selected.select-arrow-active:after {
+      top: 12px;
+      border-color: currentColor transparent transparent transparent;
+    }
+    #theme-selected,
+    #themes div {
+      padding: 5px 8px;
+      border: 1px dotted transparent;
+      cursor: pointer;
+    }
+    #theme-selected:hover,
+    #theme-selected.select-arrow-active {
+      border-color: currentColor
+    }
+    #themes {
+      position: absolute;
+      bottom: 100%;
+      left: 0;
+      right: 0;
+      height: 218px;
+      overflow-y: auto;
+      z-index: 97;
+      border: 1px dotted;
+    }
+    #themes div:hover,
+    .same-as-selected {
+      background-color: rgba(0, 0, 0, 0.1);
+    }
   </style>
 </head>
 
 <body>
   <%= yield %>
   <footer id="page-footer">
-    <small>Rendered with theme
-      <strong style="text-transform: capitalize">
-        <%= @theme.class.name.gsub(/_/, ' ') %>
-      </strong>
-    </small>
+    <small>Rendered with theme:</small>
+    <div id="theme-selector">
+      <select>
+        <% Rouge::Theme.registry.each_key do |name|
+           %><option value="<%= name %>"<% if @theme.class.name == name %> selected<% end %>><%= name %></option>
+        <% end %>
+      </select>
+    </div>
   </footer>
+
+  <script>
+    /*
+     * --------------------------------------------------------------
+     *   Adapted from sample code found at:
+     *     https://www.w3schools.com/howto/howto_custom_select.asp
```

Comment:
Do we really need a custom selector? This isn't a fancy consumer-facing site? A regular HTML5 selector should be enough for our purposes.